content/blog/2020/06/2020-06-12-infra-and-aws-donation.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "AWS and Infrastructure Updates - Hello from Infra"
+tags:
+- aws
+- community
+- infrastructure
+- kubernetes
+author: olblak
+opengraph:
+  image: /images/post-images/2020-06-12-infrastructure-opengraph.png
+---
+
+image:/images/post-images/2020-06-12-infra-update/infrastructure-opengraph.png[World map,role="center", float="right", height=231]
+
+During April 2020:
+
+* Over 600 000 people visited link:https://www.jenkins.io/[www.jenkins.io]
+* Over 250 000 Jenkins servers regularly checked the
+  link:https://pkg.jenkins.io/[Jenkins package server] and the
+  link:https://updates.jenkins.io/[Jenkins update server]
+* Over 43 000 continuous integration jobs ran on link:https://ci.jenkins.io/[ci.jenkins.io]
+* Over 950 plugins ran their continuous integration pipelines on link:https://ci.jenkins.io/[ci.jenkins.io]
+
+All those services require infrastructure.
+The Jenkins project relies heavily on its infrastructure.
+We use websites like link:/[www.jenkins.io] and link:https://plugins.jenkins.io/[plugins.jenkins.io], ticketing systems like link:https://issues.jenkins.io/[issues.jenkins.io]  , CI/CD infrastructure like link:https://ci.jenkins.io/[ci.jenkins.io] , and many other services.
+And like any organization, there are specific people who make sure that those services are always up and running.
+
+The Jenkins project, as an open source project, is built and maintained by its awesome community of users.
+Everyone is welcome to participate.
+Infrastructure is no exception.
+
+We have a vision to be a community effort trying to be as transparent as possible.
```

Comment:
```suggestion
While we can’t share publicly everything like secrets, certificates,…​ 
we still try to be as transparent as possible so that everybody can understand and improve our infrastructure without having privileged accesses
```

Took from https://www.jenkins.io/projects/infrastructure/#contributing